app/src/cli/new_package.rs

```diff
@@ -48,7 +48,11 @@ pub(crate) fn handle(
     // Write the files to disk.
     let ambient_toml = include_str!("new_package_template/ambient.toml")
         .replace("{{id}}", id.as_str())
-        .replace("{{name}}", name);
+        .replace("{{name}}", name)
+        .replace(
+            "{{ambient_version}}",
+            &format!("{}", ambient_native_std::ambient_version().version),
```

Comment:
Maybe move out this `ambient_version` call upwards and use it for both this and `cargo_toml`?